### ISSUE TITLE
Not Starting with WINDOWS

### ISSUE THREAD

**Issue by Rutilicus33** [Rem0o/FanControl.Releases#478]
FanControl doesn't start automatically with windows even if it is configured in the program itself and also added to the shell:startup.

With the next windows start the configuration tick of the program "Start with Windows" changes to "Not Selected" automatically

**Comment by 3rbium** [comment]
Just checking that you're a local admin of the PC? I tried everything I could to get auto-start working as a standard user, but nothing helped. Ended up having to change my user to Admin as found FanControl so useful!

**Comment by Rem0o** [comment]
FanControl indeed only works on admin users.

**Comment by stavrosthewonderchicken** [comment]
Can confirm last 3 or 4 versions do not start automatically with Windows when previous versions did.

**Comment by Rem0o** [comment]
If you take a look in the Task Scheduler for the task named "Fan Control", it is created properly? Can you run it manually?

**Comment by stavrosthewonderchicken** [comment]
Sure -- running it manually does not start FanControl. Here's what it looks like in situ:

![image](https://user-images.githubusercontent.com/1720914/150867899-daba4933-df68-4d50-ba89-719cb05a896b.png)

**Comment by Rem0o** [comment]
Wait, how can it has never run?  
Also, can you just run the command line in the FanControl folder and see what's wrong?

**Comment by stavrosthewonderchicken** [comment]
That’s what I was wondering. Also, I’m in the office now so won’t be able to check the command line until later...

**Comment by stavrosthewonderchicken** [comment]
> Also, can you just run the command line in the FanControl folder and see what's wrong?

Sorry, do you mean Unregister_Startup_Task.cmd?

**Comment by Rem0o** [comment]
The command from the task in the task scheduler.

**Comment by stavrosthewonderchicken** [comment]
If I run the command ( start /B FanControl.exe) from a prompt in the program folder without the /c switch, it starts FanControl OK.

![image](https://user-images.githubusercontent.com/1720914/150943267-a243950e-00f8-436d-ac1d-e613f754c9bd.png)

Thought this was odd, though.

![image](https://user-images.githubusercontent.com/1720914/150943519-902f096c-1941-4365-a421-e3e74aa07379.png)

**Comment by stavrosthewonderchicken** [comment]
Huh. Tries to change it to Windows 10 and save the task and got an error message:

![image](https://user-images.githubusercontent.com/1720914/150944058-b0265396-47bd-4e9a-9341-9bbcd7eb8c0f.png)

**Comment by Rem0o** [comment]
What's your user account(s) situation?

**Comment by Rutilicus33** [comment]
I am using admin account from the beginning and I cannot make the FanControl start with windows, no matter what I'very tried.

**Comment by Rem0o** [comment]
I'm asking because I see "Medium Mandatory Level" as a user account, thought that was a strange name and it might not be the admin one.

**Comment by stavrosthewonderchicken** [comment]
Admin account, same as forever, no changes there.

**Comment by Rem0o** [comment]
Well I'm really out of ideas here. The task is there, its command work, admin rights and everything. 
Sorry 🤷‍♂️

**Comment by stavrosthewonderchicken** [comment]
Something seems misconfigured in the task scheduler item, though, right? I mean if it won't let me save the item after opening it (even if I make no changes other than add a period to the end of the description phrase)...

**Comment by Rem0o** [comment]
You can't change the Windows Vista thing. That's basically the versioning of the task, saying how far back it is retro-compatibility.

**Comment by Rutilicus33** [comment]
I am using Avira antivirus and malwarebytes which quarantined FanControl.exe after my first use of the software. I don't know whether it has something to do or not.

**Comment by stavrosthewonderchicken** [comment]
> You can't change the Windows Vista thing. That's basically the versioning of the task, saying how far back it is retro-compatibility. That was fine don't change it.

That's not what I'm saying - what I mean is you can't anything, even a label (see screesnot) without it refusing to save the task again. So something's not right there, clearly.

![image](https://user-images.githubusercontent.com/1720914/151063782-5ed14dc9-ffa2-4eca-8372-c2d33a582c7f.png)

**Comment by Rem0o** [comment]
Try deleting it all together, go back in FanControl and tick the box again, it will recreate it from scratch.

**Comment by stavrosthewonderchicken** [comment]
Will do when I get home from work and will report back...

**Comment by stavrosthewonderchicken** [comment]
OK, so I 

1) deleted the Task Scheduler entry.
2) Ran Fan Control manually, set it to start with Windows, saved config.
3) Checked that new Task had been created - it had.
4) Rebooted.
5) No autostart still.

Just to see if anything had changed, I added a couple of characters to the description of the task item as above, and it once again would not save, and threw the same error message.

**Comment by alextruuvert** [comment]
> I am using Avira antivirus and malwarebytes which quarantined FanControl.exe after my first use of the software. I don't know whether it has something to do or not.

Just now I found that Malwarebytes had quarantined some tasks related to FanControl (most of the quarantine list didn't include "fancontrol" in the name, but were also needed so be careful allowing some of those quarantined items).


Un-quarantining those items within Malwarebytes brought back the Start with Windows function for me.

**Comment by stavrosthewonderchicken** [comment]
FWIW I use no antivirus other than Windows built-in.

**Comment by Rutilicus33** [comment]
> > I am using Avira antivirus and malwarebytes which quarantined FanControl.exe after my first use of the software. I don't know whether it has something to do or not.
> 
> Just now I found that Malwarebytes had quarantined some tasks related to FanControl (most of the quarantine list didn't include "fancontrol" in the name, but were also needed so be careful allowing some of those quarantined items).
> 
> Un-quarantining those items within Malwarebytes brought back the Start with Windows function for me.

I have also un-quarantined FanControl, but even it is not starting automatically :(

**Comment by Rutilicus33** [comment]
I have un-quarantined the FanControl folder, but it still doesn't start with windows :(

**Comment by Tabascl** [comment]
I solved the issue by doing the following steps:

1. Change the user to your usual account name in security options
2. Make sure to only execute if the user is logged in
3. Change the trigger from any user to your usual account

**Comment by Rutilicus33** [comment]
Ich am using only one account name, so it is still not working for me.

**Comment by ilovenobody123** [comment]
I have this problem too. I do have Malwarebytes, but it isn’t eating any task or files for FanControl. Anyone got it working? Very awesome tool, hope I get it working again.

**Comment by xhul-dev** [comment]
> I have this problem too. I do have Malwarebytes, but it isn’t eating any task or files for FanControl. Anyone got it working? Very awesome tool, hope I get it working again.
> 
> Edit: Apologies, commented too early, I ended up changing the command and directing the task to run the .exe. Seems to have worked for now.

Otherwise, you can also use the tool i created for the occasion: https://github.com/xhul-dev/FanControl-WinStart
It creates a task that's an exact match with the one Fan Control creates.
Cheers.

**Comment by mooock** [comment]
normally if you cannot save a task, its because you have a user with auto login. try to type the same username in the login box and save again, and it now should work

**Comment by Rem0o** [comment]
It works as long as the selected user is admin. It always worked this way for FanControl.

**Comment by vilyamss** [comment]
The FanControl program creates the wrong task in the startup scheduler, in the place of the program path in the action parameter there is just the name of the program, I wrote the full path to the file there and everything worked.
![Безымянный](https://github.com/user-attachments/assets/9c0d09e5-fd68-4745-9cd5-0472c60100e3)

**Comment by merlin-tc** [comment]
@vilyamss Thanks, that fixed it for me!

This fixed it for me. Somehow you can start it while logged in and in the task scheduler but the same job does not work when the system starts.


> The FanControl program creates the wrong task in the startup scheduler, in the place of the program path in the action parameter there is just the name of the program, I wrote the full path to the file there and everything worked.

**Comment by myusernamerulez** [comment]
I had the same issue now with v251. I was using an old version from 2022 beforehand, that was starting fine with Windows ever since its first install.
When I installed this v251, it didn't start automatically with Windows10.

So what I did was that changed the permissions on the fancontrol.exe to enable the users everything, then also had to add the respective autorun entry to the registry: "E:\fancontrol2025\fcshortcut.lnk"

The shortcut was created using the hint above from [stavrosthewonderchicken], so the shortcut's Target is C:\Windows\System32\cmd.exe /C start /B FanControl.exe

Now FanControl does start when I log on.
